static-site/FAQ.md

```diff
@@ -85,65 +98,50 @@ While the type of resources worth reviewing when onboarding to the DASG ecosyste
-They are specific to the CCW and are not applicable to any other identification system or data source. 
+The [Claim Number](https://resdac.org/cms-data/variables/claim-number) found in the [CMS Portal](MyMedicare.gov) ("check my claims") is the IDR (Integrated Data Repository) Claim Unique ID and is derived from data in the National Claims History (NCH) file process. 
+The [Claim ID](https://resdac.org/cms-data/variables/claim-id) (CLM_ID) and [Claim Group ID](https://bluebutton.cms.gov/resources/identifier/claim-group/) is specific to the CCW and is not applicable to any other identification system or data source. 
 
-#### Why is the Provider NPI (National Provider Identifier) sometimes an invalid number, such as 9999999992, 9999999993. Similarly, why is the Tax Number sometimes 11111111111?
+The 'Claim Group ID' value is derived within the process from loading CCW data into the BFD database. 
 
-The upstream processing systems (MCS (Multi-Carrier System) or FISS (Fiscal Intermediate Shared System)) sometimes have strange default values like this that get passed on to CCW, and then subsequently to BFD.
+#### Why is the Provider NPI (National Provider Identifier) sometimes an invalid number, such as 9999999992, 9999999993. Similarly, why is the Tax Number sometimes 11111111111?
 
-BFD, in most cases, does not correct this type of data passed to it from upstream providers. BFD seeks to enrich the data that is provided.
+The upstream processing systems, MCS (Multi-Carrier System) or FISS (Fiscal Intermediary Shared System), sometimes have default values that get passed on to CCW, and subsequently, to BFD.
 
 #### Are pending or partially adjudicated claims available? How long does it take for those claims to be available?
 
-Yes, this data is provided by RDA (Replicated Data Access API) and available within BFD from the Claim and ClaimResponse resources.
+Yes,  PAC data comes from FISS (Fiscal Intermediary Shared Systems) and MCS (Multi Carrier Systems)  via the MPSM (Medicare Payment System Modernization) RDA (Replicated Data Access) API and is made available in BFD  through the Claim and ClaimResponse resources.
+
 
-The amount of time it takes a claim to progress through the data ecosystem depends on a variety of factors.
-A shared CMS [whitepaper](https://confluence.cms.gov/display/BFD/FHIR+and+CMS+Documentation+Links?preview=/415007408/455427981/medicare-claims-maturity%20(1).pdf) documents some additional details on the claim lifecycle.
+PAC data is received from the MPSM RDA API  as it becomes available, typically nightly. <ins>However, it is important to note that PAC data availability varies by data source.</ins> This shared CMS [whitepaper](https://confluence.cms.gov/display/BFD/FHIR+and+CMS+Documentation+Links?preview=/415007408/455427981/medicare-claims-maturity%20(1).pdf) documents some additional details on the claim lifecycle.
```

Comment:
The markdown renderer that Jekyll uses isn't the most feature-rich and I'm unsure if `<ins>` will render correctly. We can deploy it and check.

Reply:
This does appear to render correctly, so we're good.